CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 **New Features**
 
 * [#40](https://github.com/dblock/resourcelib/pull/40): Added `Resource.Save(string filename, IEnumerable<Resource> resources)` for saving multiple resources in one batch - [@thoemmi](https://github.com/thoemmi).
+* [#46](https://github.com/dblock/resourcelib/pull/46): Added set-method for `Accelerator.Keys`, 
```

Comment:
What's up with new lines and plusses ;) 

Make this three entires with the same pull request number and your name.

Added set method for for `Accelerator.Keys`
Added `Accelerator.Flags`
Added `Accelerator.addFlag(User32.AcceleratorVirtualKey)` and `Accelerator.removeFlag(User32.AcceleratorVirtualKey)`